assets/scss/6-components/block-grid/_block-grid.scss

```diff
@@ -0,0 +1,25 @@
+// Block grid (c-block-grid)
+//
+// Auto flow grid of child elements. Mobile has a set rule to stack, while larger viewports auto flow based on min/max width rule on child elements. If the container does not allow for each column to be at least 200px, remaining columns will wrap to the next row. {{isWide}}
+//
+// .c-block-grid--bordered - Has a top/bottom border for separation
+//
+//
+// Markup: 6-components/block-grid/block-grid.html
+//
+// Styleguide 6.1.3
+.c-block-grid {
+  @include gap($size-s);
+  display: grid;
+
+  @include mq($from: bp-m) {
+    grid-template-columns: repeat(auto-fit, minmax(100px, 1fr));
```

Comment:
Should this be 200px?

Reply:
Good catch. In the homepage revamp, I actually did want 100px, but my comment above that line still references 200px. Fixed!